catalog-rest-service/src/main/java/org/openmetadata/catalog/CatalogApplication.java

```diff
@@ -97,6 +99,9 @@ public void run(CatalogApplicationConfig catalogConfig, Environment environment)
     final Jdbi jdbi = new JdbiFactory().build(environment, catalogConfig.getDataSourceFactory(), "database");
     jdbi.setTimingCollector(new MicrometerJdbiTimingCollector());
 
+    final SecretsManager secretsManager =
```

Comment:
should we make this as singleton that way we don't need to worry about passing it around?